solutions/usaco-761.mdx

```diff
@@ -0,0 +1,94 @@
+---
+id: usaco-761
+source: USACO Bronze 2017 December
+title: Milk Measurement
+author: Ryan Chou
+---
+
+[Official Analysis (C++ and Java)](http://www.usaco.org/current/data/sol_measurement_bronze_dec17.html)
+
+## Explanation
+
+Since there are only $3$ cows and $N \leq 1000$, we can sort all of the entries by date and update the selected cow's milk value for each entry.
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N)$
+<LanguageSection>
+<PySection>
+
+```py
+import sys
+sys.stdin = open('measurement.in', 'r')
+sys.stdout = open('measurement.out', 'w')
+
+n = int(input())
+INIT_MILK = 7
+logs = []
+# Every cow starts off producing 7 gallons of milk each.
+bessie, elsie, mildred = INIT_MILK, INIT_MILK, INIT_MILK
+
+"""
+Since all cows produce the same amount of milk,
+they'll all be on the display at the beginning.
+"""
+display = {'b', 'e', 'm'}
+
+"""
+This will store the past display,
+so we can judge whether FJ needs to change his display.
+"""
+tmp_display = {}
+ans = 0
+for _ in range(n):
+	time, name, addition = input().split()
+	logs.append([int(time), name, int(addition)])
+
+logs.sort()
+for i in logs:
+	if i[1] == 'Bessie':
+		bessie += i[2]
+	elif i[1] == 'Elsie':
+		elsie += i[2]
+	elif i[1] == "Mildred":
+		mildred += i[2]
+
+	# Find the maximum amount of milk that's being produced.
+		curmax = max(bessie, max(elsie, mildred))
```

Comment:
uh what happened here

Reply:
wait need to change github's indents from 8 spaces to 4 spaces

Reply:
still seems to be weird?

Reply:
🤔

Reply:
![image](https://user-images.githubusercontent.com/55369003/127720875-aec38396-4326-4bdf-be4f-5bd3f9b14111.png)

Reply:
it looks fine in the editor?
![image](https://user-images.githubusercontent.com/81596991/127720937-66691d17-119a-4781-80f3-f67962758dc4.png)

Reply:
i also indented the comment (that might have been the problem)

Reply:
wait is that curmax supposed to be in the elif?

Reply:
no